app/src/pages/[resourceType]/[id].tsx

```diff
@@ -61,6 +65,15 @@ export default function ResourceIDPage({ jsonData }: InferGetServerSidePropsType
               </Prism>
             </Tabs.Panel>
           )}
+          {decodedELM != null && (
+            <Tabs.Panel value="elm" pt="xs">
+              {/* eslint-disable  @typescript-eslint/no-explicit-any */}
+              <Prism language="json" colorScheme="light">
+                {/* eslint-enable  @typescript-eslint/no-explicit-any */}
```

Comment:
Now that we are properly using `json` as the language, we can get rid of these comments that disable the lint rules for no explicit any.